andela/bad_ass-ah-frontend
# #166266972 Update and delete comment

**What does this PR do?**

- This PR will allow users to update and delete their comments

**Description of Task to be completed?**

- Given user will be able to delete their comments
- Given user will be able to update his comments

**How should this be manually tested?**

- Ensure that the server is running or run it with the command `npm run dev`

- Make sure that you are logged in the app

- From the `Homepage` click on a given article

- then post a comment

- Then hover the three right button and click respectively to `delete` or `update`

- If editing, finally click on the `save changes` button after editing the comment


**What are the relevant pivotal tracker stories?**

[#166266972](https://www.pivotaltracker.com/story/show/166266972)

**Screenshots (if appropriate)**

![image](https://user-images.githubusercontent.com/46047244/59163286-301cd400-8aff-11e9-82e8-4138590941a3.png)

![image](https://user-images.githubusercontent.com/46047244/59163290-37dc7880-8aff-11e9-99d4-4a63f0998f07.png)

![image](https://user-images.githubusercontent.com/46047244/59163293-3e6af000-8aff-11e9-85b2-92153d7b5f6b.png)

![image](https://user-images.githubusercontent.com/46047244/59163295-44f96780-8aff-11e9-9240-2ee2c64a88ed.png)

![image](https://user-images.githubusercontent.com/46047244/59163298-49be1b80-8aff-11e9-8a00-10528164ba31.png)

![image](https://user-images.githubusercontent.com/46047244/59163299-4f1b6600-8aff-11e9-8689-ffe723316fc6.png)

![image](https://user-images.githubusercontent.com/46047244/59163301-53e01a00-8aff-11e9-8a3d-7b43e941a3eb.png)

![image](https://user-images.githubusercontent.com/46047244/59163302-580c3780-8aff-11e9-9dae-f228b81c3d21.png)
